### PR TITLE
Fix Colorize doc example

### PR DESCRIPTION
**Pull request by zw963**

### CODE REVIEW

**Review by straight-shoota — APPROVED** [review]

**Comment by zw963** [comment]
If we should always merge those document fix and add it into  next release ASAP ? 

e.g. #12343, and current PR.

My concern is,  those document fix never breaking anything, right? those a very simple changes, we can always release it safely, not worry about breaking something, probably we need a individual more efficient/simple CI process for those simple doc things instead.

We have a lot of documentation need additions/fixes/improvements, better documentation is better for newcomers, i really want to help, but the current process leaves me unmotivated.

@straight-shoota  @asterite @Blacksmoke16

**Comment by straight-shoota** [comment]
Yes, this PR is a trivial fix and it should be fine to accept it without a second approval.

https://github.com/crystal-lang/crystal/pull/12343 on the other hand is not as trivial and I think it's important to vet such documentation changes, even if it won't break any code.

It's still a very simple change, so reviewing should be pretty quick. I do not know why no other Core Team member has approved it yet.

**Review by oprypin — APPROVED** [review]

**Pull request merged**
